app/src/main/java/org/mozilla/focus/fragment/onboarding/FinishOnboardingInteractor.kt

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.fragment.onboarding
+
+import android.content.Context
+import androidx.preference.PreferenceManager
+import org.mozilla.focus.state.AppAction
+import org.mozilla.focus.state.AppStore
+
+class FinishOnboardingInteractor(val appStore: AppStore) {
```

Comment:
Although the screen has only one available action at the moment, I would prefer to have this interactor named similarly to other interactors and the possible actions declared separately. (even if it is only one at the moment)